bgpd/bgp_vty.c

```diff
@@ -12244,6 +12244,7 @@ lcommunity_list_set_vty (struct vty *vty, int argc, struct cmd_token **argv,
       return CMD_WARNING;
     }
 
+  idx = 0;
```

Comment:
Don't "AA:BB:CC" and "LINE" always come after the other parameters? It's not clear to me what this patch is fixing.